camel-openmrs/src/main/java/org/openmrs/eip/component/entity/BaseEntity.java

```diff
@@ -9,19 +9,25 @@
 @MappedSuperclass
 public abstract class BaseEntity {
 
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    @Column(name = "id")
-    private Long id;
-
-    @NotNull
-    @Column(name = "uuid")
-    private String uuid;
-
-    /**
-     * Tests if an entity is out of date compared the given entity
-     * @param entity entity to test
-     * @return true if out of date
-     */
-    public abstract boolean wasModifiedAfter(BaseEntity entity);
+	@Id
```

Comment:
What did you change? If nothing please revert

Reply:
I will revert this, wasn't my intention to commit that.